scripts/gulpfiles/build_tasks.js

```diff
@@ -15,6 +15,7 @@ gulp.sourcemaps = require('gulp-sourcemaps');
 
 const path = require('path');
 const fs = require('fs');
+const os = require('os');
```

Comment:
Initially I wanted to use `process.platform` but it seems that os.platform is slightly [more reliable](https://stackoverflow.com/a/61744810/529024). since `os` is a native module I thought it should effect the size or the performance of the build. I am happy to change it if that is no the case.